api/host.go

```diff
@@ -32,7 +32,7 @@ type (
 	// to /host/storage - a bunch of information about the status of storage
 	// management on the host.
 	StorageGET struct {
-		Folders []modules.StorageFolderMetadata `json:"folders"`
+		FolderMetadata []modules.StorageFolderMetadata `json:"foldermetadata"`
```

Comment:
I think it's advantageous to have json tags that are one word long. Consider the hostdb endpoints that return an object with a single "host" field.